crates/egui_glow/src/shader/fragment.glsl

```diff
@@ -1,6 +1,12 @@
 #ifdef GL_ES
+// To avoid weird distortion issues when rendering text etc, we want highp if possible.
+// But apparently some devices don't support it, so we have to check first.
+#if defined(GL_FRAGMENT_PRECISION_HIGH) && GL_FRAGMENT_PRECISION_HIGH == 1
+    precision highp float;
+#else
     precision mediump float;
 #endif
+#endif
```

Comment:
```suggestion
#ifdef GL_ES
    // To avoid weird distortion issues when rendering text etc, we want highp if possible.
    // But apparently some devices don't support it, so we have to check first.
    #if defined(GL_FRAGMENT_PRECISION_HIGH) && GL_FRAGMENT_PRECISION_HIGH == 1
        precision highp float;
    #else
        precision mediump float;
    #endif
#endif
```

Reply:
NOTE: I cannot commit these changes directly, because you opened your PR from your `master`.

Reply:
This is mentioned in the PR template, but since most people seem to only skim that (which I understand), I created a PR to enforce it in the future:
* https://github.com/emilk/egui/pull/6899

Reply:
Done.